app/api/v1/endpoints/admin_testing.py

```
@@ -0,0 +1,169 @@
+    if not current_user.is_admin():
+        raise HTTPException(status_code=403, detail="Admin access required")
+    
+    # Get all strategy functions via introspection
+    all_functions = []
+    
+    for attr_name in dir(trading_strategies_service):
+        attr = getattr(trading_strategies_service, attr_name)
+        
+        if (callable(attr) and 
+            not attr_name.startswith('_') and 
+            not attr_name in ['logger', 'log', 'async_init', 'cleanup'] and
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)